Appserver/FormSubmit/MileageForm.cs

```diff
@@ -0,0 +1,35 @@
+﻿using Newtonsoft.Json;
+using System.Collections.Generic;
+
+namespace Appserver.FormSubmit
+{
+    public class MileageForm : AbstractFormObject
+    {
+        private List<MileageRowItem> miles = new List<MileageRowItem>();
+        public MileageForm() { }
+        public int id { get; set; }
+        public string clientName { get; set; }
+        public string prime { get; set; }
+        public string providerName { get; set; }
+        public string providerNum { get; set; }
+        public string brokerage { get; set; }
+        public string scpaName { get; set; }
+        public string serviceAuthorized { get; set; }
+
+        [JsonProperty("mileagesheet")]
```

Comment:
PWA currently has this field as 'serviceDeliveredOn'. PWA will change it to mileagesheet.

Reply:
'authorized' is a string that contains the provider initials, it is not a bool